core/ui/src/components/settings/CloudLogManagerConfigureModal.vue

```diff
@@ -0,0 +1,326 @@
+    onModalHidden() {
+      this.$emit("hide");
+    },
+    resetModal() {
+      this.radioVal = "last_timestamp";
+      this.toggleEnabled = false;
+      this.address = "";
+      this.tenant = "";
+      this.date = "";
+      this.time = "";
+      this.last_timestamp_formatted = "";
+    },
+    goToMoreDetails() {
+      window.open(
+        "https://www.nethesis.it/servizi/cloud-log-manager",
+        "_blank"
+      );
+    },
+    onModalShown() {
+      if (
+        this.configuration.status == "active" ||
+        this.configuration.status == "failed"
+      ) {
+        this.toggleEnabled = true;
+      }
+
+      if (this.configuration.address == "") {
+        this.address = "https://nar.nethesis.it";
+      } else {
+        this.address = this.configuration.address;
+      }
+
+      if (this.configuration.tenant == "") {
+        this.tenant = "clienteit";
+      } else {
+        this.tenant = this.configuration.tenant;
+      }
+
+      if (this.configuration.last_timestamp == "") {
+        this.radioVal = "choose_date_time";
+      } else {
+        const last_timestamp_date = new Date(this.configuration.last_timestamp);
+        this.last_timestamp_formatted = this.formatDate(
+          last_timestamp_date,
+          "Pp"
+        );
+      }
+
+      this.date = this.formatDate(new Date(), "yyyy-MM-dd");
+      this.time = "00:00";
+    },
+    validateCloudLogManagerInput() {
+      this.clearErrors(this);
+      let isValidationOk = true;
+
+      if (!this.address) {
+        this.error.address = this.$t("common.required");
+        isValidationOk = false;
+      }
+      if (!this.tenant) {
+        this.error.tenant = this.$t("common.required");
+        isValidationOk = false;
+      }
+
+      return isValidationOk;
+    },
+    async setCloudLogManager() {
+      if (!this.validateCloudLogManagerInput()) {
+        return;
+      }
+
+      this.loading.setCloudLogManager = true;
+      this.error.setCloudLogManager = "";
+      const taskAction = "set-clm-forwarder";
+
+      // register to task abortion
+      this.$root.$once(taskAction + "-aborted", this.setCloudLogManagerAborted);
+
+      // register to task completion
+      this.$root.$once(
+        taskAction + "-completed",
+        this.setCloudLogManagerCompleted
+      );
+
+      if (this.radioVal == "choose_date_time") {
+        this.start_time = new Date("".concat(this.date, " ", this.time));
+      } else {
+        this.start_time = "";
+      }
+
+      if (this.toggleEnabled) {
+        this.active = true;
+      } else {
+        this.active = false;
+      }
+
+      const res = await to(
+        this.createModuleTaskForApp(this.lokiToEdit.instance_id, {
+          action: taskAction,
+          data: {
+            active: this.active,
+            address: this.address,
+            tenant: this.tenant,
+            start_time: this.start_time,
+          },
+          extra: {
+            isNotificationHidden: true,
```

Comment:
`isNotificationHidden: true` should be used only for read actions. Use this as reference: https://github.com/NethServer/ns8-core/blob/main/core/ui/src/components/nodes/SetFqdnModal.vue#L218-L220